airflow/www/extensions/init_views.py

```diff
@@ -73,8 +73,14 @@ def init_plugins(app):
     appbuilder = app.appbuilder
 
     for view in plugins_manager.flask_appbuilder_views:
-        log.debug("Adding view %s", view["name"])
-        appbuilder.add_view(view["view"], view["name"], category=view["category"])
+        try:
+            log.debug("Adding view %s with menu", view["name"])
```

Comment:
What do you think should be the behavior when the user passes "None" in the "name" key?

Reply:
Hmm.. It's technically a valid string, I doubt the user would expect that to actually mean None.
It might be more possible to user to pass an empty string though.

Reply:
Handled the case for empty string now. Do you suppose I should check for "None" value too?